test_runner/performance/test_perf_olap.py

```diff
@@ -88,7 +108,10 @@ def get_scale() -> List[str]:
     return [scale]
 
 
-def run_psql(env: RemoteCompare, labelled_query: LabelledQuery, times: int) -> None:
+# run the query times times plus once with EXPLAIN VERBOSE if TEST_OLAP_COLLECT_EXPLAIN is set to true (default false)
```

Comment:
This comment might be a bit outdated, we don't really check `TEST_OLAP_COLLECT_EXPLAIN` here anymore

Reply:
fixed